pkg/controller/cronjob/cronjob_controller_test.go

```diff
@@ -248,113 +248,115 @@ func TestSyncOne_RunOrNot(t *testing.T) {
 		"prev ran but done, long overdue, past short deadline, F":  {f, F, onTheHour, shortDead, T, F, weekAfterTheHour(), T, F, 1, 0},
 	}
 	for name, tc := range testCases {
-		sj := cronJob()
-		sj.Spec.ConcurrencyPolicy = tc.concurrencyPolicy
-		sj.Spec.Suspend = &tc.suspend
-		sj.Spec.Schedule = tc.schedule
-		if tc.deadline != noDead {
-			sj.Spec.StartingDeadlineSeconds = &tc.deadline
-		}
-
-		var (
-			job *batchv1.Job
-			err error
-		)
-		js := []batchv1.Job{}
-		if tc.ranPreviously {
-			sj.ObjectMeta.CreationTimestamp = metav1.Time{Time: justBeforeThePriorHour()}
-			sj.Status.LastScheduleTime = &metav1.Time{Time: justAfterThePriorHour()}
-			job, err = getJobFromTemplate(&sj, sj.Status.LastScheduleTime.Time)
-			if err != nil {
-				t.Fatalf("%s: nexpected error creating a job from template: %v", name, err)
-			}
-			job.UID = "1234"
-			job.Namespace = ""
-			if tc.stillActive {
-				sj.Status.Active = []v1.ObjectReference{{UID: job.UID}}
-				js = append(js, *job)
-			}
-		} else {
-			sj.ObjectMeta.CreationTimestamp = metav1.Time{Time: justBeforeTheHour()}
-			if tc.stillActive {
-				t.Errorf("%s: test setup error: this case makes no sense", name)
+		t.Run(name, func(t *testing.T) {
```

Comment:
Future nitpick: it would have been good to do the `t.Run` change as a separate commit. Kubernetes PRs are typically tough to review, and mixing any refactoring/cleanup with the logic changes adds to that. 😅

Reply:
it's easy enough to just do it -> split out the `t.Run` change into https://github.com/lyft/kubernetes/pull/17